apollo-api/src/main/java/com/apollographql/android/api/graphql/Field.java

```diff
@@ -148,7 +148,9 @@ private String orderIndependentKeyForVariableArgument(Map<String, Object> object
     Object variable = objectMap.get(VARIABLE_NAME_KEY);
     //noinspection SuspiciousMethodCalls
     Object resolvedVariable = variables.valueMap().get(variable);
-    if (resolvedVariable instanceof Map) {
```

Comment:
maybe I'm missing smth but in case if `resolvedVariable ` is null this will always be false without NPE: `if (null instanceof Map)`